pycine/file.py

```diff
@@ -55,20 +83,16 @@ def write_header(cine_file, header, backup=True):
     with open_ignoring_read_only(cine_file, "rb+") as f:
         f.write(header["cinefileheader"])
         f.write(header["bitmapinfoheader"])
-        f.seek(header["cinefileheader"].OffSetup)
```

Comment:
Why is this removed?

Reply:
in fact I don't know, why I don't had it in my version. It seems to work like this too. 
In the file docu (version 2020) there is no gap between the BITMAPINFOHEADER and the SETUP part, so it should be fine without the seek(). But for savety reasons this can be left in.
I have added it in a new commit